nnsmith/backends/torchjit.py

```diff
@@ -0,0 +1,64 @@
+import os
+import warnings
+from typing import Dict, Tuple
+
+import numpy as np
+import torch
+from multipledispatch import dispatch
+from torch.utils.mobile_optimizer import optimize_for_mobile
+
+from nnsmith.backends.factory import BackendCallable, BackendFactory
+from nnsmith.materialize.torch import TorchModel
+
+# Check https://pytorch.org/tutorials/recipes/recipes/tuning_guide.html
+# for more PyTorch-internal options.
+TORCH_JIT_OPT_MOBILE = os.getenv("TORCH_JIT_OPT_MOBILE", "0") == "1"
```

Comment:
nit: Always use `NNSMITH_` prefix as a namespace to avoid possible env var conflicts. But for conciseness, we may consider `NNSMITH_PTJIT_OPT_MOBILE`.